dojo/api_v2/serializers.py

```diff
@@ -388,6 +390,65 @@ class BurpRawRequestResponseSerializer(serializers.Serializer):
     req_resp = RequestResponseSerializerField(required=True)
 
 
+class BurpRawRequestResponseMultiSerializer(serializers.ModelSerializer):
+    burpRequestBase64 = serializers.CharField()
+    burpResponseBase64 = serializers.CharField()
+
+    def get_stringrequest(self, obj) -> str:
+        result = ""
+        if isinstance(obj, dict):
+            result = obj.get("string_request", "")
+        elif obj.string_request:
+            result = obj.string_request
+
+        return result
+
+    def get_stringresponse(self, obj) -> str:
+        result = ""
+        if isinstance(obj, dict):
+            result = obj.get("string_response", "")
+        elif obj.string_response:
+            result = obj.string_response
+
+        return result
+
+    def to_representation(self, data):
+        return {
+            "id": data.id,
+            "finding": data.finding.id,
+            "burpRequestBase64": data.burpRequestBase64.decode("utf-8"),
+            "burpResponseBase64": data.burpResponseBase64.decode("utf-8"),
+        }
+
+    def validate(self, data):
+        b64request = data.get("burpRequestBase64", None)
+        b64response = data.get("burpResponseBase64", None)
+        finding = data.get("finding", None)
+
+        if not b64request or not b64response or not finding:
+            msg = "burpRequestBase64, burpResponseBase64, and finding are required."
+            raise ValidationError(msg)
+
+        base64.b64decode(b64request, validate=True)
+        base64.b64decode(b64response, validate=True)
+        b64request = b64request
+        b64response = b64response
+        data["burpRequestBase64"] = b64request.encode("utf-8")
+        data["burpResponseBase64"] = b64response.encode("utf-8")
+        if finding and b64request and b64response:
+            BurpRawRequestResponse(finding=finding,
+                                   burpRequestBase64=b64request,
+                                   burpResponseBase64=b64response).clean()
+        else:
+            msg = "Failed to validate data. finding, burpRequestBase64, and burpResponseBase64 cannot be null"
+            raise ValidationError(msg)
+        return data
```

Comment:
```suggestion
        # Make sure all fields are present
        if not b64request or not b64response or not finding:
            msg = "burpRequestBase64, burpResponseBase64, and finding are required."
            raise ValidationError(msg)
        # Verify we have true base64 decoding
        try:
            base64.b64decode(b64request, validate=True)
            base64.b64decode(b64response, validate=True)
        except Exception as e:
            msg = "Inputs need to be valid base64 encodings"
            raise ValidationError(msg) from e
        # Encode the data in utf-8 to remove any bad characters
        data["burpRequestBase64"] = b64request.encode("utf-8")
        data["burpResponseBase64"] = b64response.encode("utf-8")
        # Run the model validation - an ValidationError will be raised if there is an issue
        BurpRawRequestResponse(finding=finding, burpRequestBase64=b64request, burpResponseBase64=b64response).clean()

```
Adding some comments and clearing out unnecessary conditionals